scripts/cmake/macros.cmake

```diff
@@ -1262,11 +1296,9 @@ endmacro(set_bfm_vmm)
 #     is not in the same directory, allowing you to pass a source file with
 #     a directory. Nomrally FILENAME should still match the filename being
 #     used.
-# @param CMD_LINE_ARGS Command line arguements to be passed to the test
-#     executable by ctest
 #
 function(do_test FILENAME)
-    set(multiVal DEFINES DEPENDS SOURCES CMD_LINE_ARGS)
```

Comment:
why is this being removed? I think we should wait until the memory leaks are fixed in the eapis.

Reply:
I added this back, but I will say that this is a dirty hack. A PR should not be accepted if it has memory leaks, so the tests should fail. At a minimum, if the code is merged, it should be merged with a failure so that it haunts us until its fixed as a memory leak is really bad (way worse than a missing unit test)

Reply:
Oh.... and I have no idea how it was removed in the first place. Some sort of merge error